AssetManagementTestProject/Asserter/Asserter.cs

```diff
@@ -15,11 +16,34 @@ public class Asserter : WebDriverAction
     public Asserter() : base()
     {
     }
+    #region URL FLUENT ASSERTION
+    public void AssertUrlEquals(string actual, string expected)
+    {
+        AssertEquals(actual, expected);
+    }
+    #endregion
+
+    #region ELEMENTS DISPLAY FLUENT ASSERTION
+    public void AssertElementIsDisplayed(string locator)
+    {
+        IsElementDisplayed(locator).Should().BeTrue();
+    }
+
+    public void AssertElementsAreDisplayed(List<string> locators)
```

Comment:
Check Mr. Dai's comments in my PR, do not just copy my code without thinking what to do with it